extensions/aws/src/main/java/io/opentelemetry/extension/aws/AwsXrayPropagator.java

```diff
@@ -190,14 +210,21 @@ public static AwsXrayPropagator getInstance() {
               + "' with value "
               + traceHeader
               + "'. Returning INVALID span context.");
```

Comment:
Is this log message still correct to say we're returning INVALID context? It is returning *an* invalid context, but is it returning *the* invalid context?

Reply:
Thanks, just removed that bit since it's sort of an implementation detail, logging the actual problem is what matters.